instrumentation/kafka-streams/README.md

```diff
@@ -68,9 +80,9 @@ Typically, there are at least two spans involved in traces produces by a Kafka S
 * One created by the Consumers that starts a Stream or Table, by `builder.stream(topic)`.
 * One created by the Producer that sends a records to a Stream, by `builder.to(topic)`
 
-By receiving records in a Kafka Streams application with Tracing enabled, the span created once
-a record is received will inject the span context on the headers of the Record, and it will get
-propagated downstream on the Stream topology. As span context is stored in the Record Headers,
+By receiving records in a Kafka Streams application with Tracing enabled, the span created, once
```

Comment:
good to revise that sentence but it still does not make so much sense. Not blocking this PR though.